src/client/components/share-button/readme.md

```diff
@@ -0,0 +1,23 @@
+# Share Button
```

Comment:
I think this component should be called `social-share`, since `share-button` implies that it's only one button, but the component does a lot more than that.